### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by newer79** [Physical-Intelligence/openpi#834]
When you were pre-training the single-arm and dual-arm data, after padding the single-arm actions to the max dimension, did you mask out the padded actions when calculating the loss and ignore their loss? I haven't seen the relevant code. Or simply calculate the loss for all the action dimensions, including the padding part.

Thank you.
